lib/widgets/profile_block_widget.dart

```diff
@@ -1,88 +1,154 @@
+                    ),
+                  ],
                 ),
-              ),
-              const SizedBox(width: 10.0),
-              Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
+              ],
+            ),
+            const SizedBox(height: 10.0),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 10.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
                 children: [
                   Text(
-                    'ZikFreeDKarteR44',
-                    style: MainStyles.kBlackColorW500(14.0),
+                    '${Strings.type} ${user.type ?? ''}',
+                    style: MainStyles.kBlackColorW500(12.0),
                   ),
                   Text(
-                    '${Strings.accountCreated} 30 april 2022',
-                    style: MainStyles.kGreyColorW500(10.0),
+                    '${Strings.publicRepos} ${user.publicRepos ?? ''}',
+                    style: MainStyles.kBlackColorW500(12.0),
                   ),
                 ],
               ),
-            ],
-          ),
-          const SizedBox(height: 10.0),
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 10.0),
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                Text(
-                  '${Strings.type} User',
-                  style: MainStyles.kBlackColorW500(12.0),
-                ),
-                Text('${Strings.publicRepos} 7', style: MainStyles.kBlackColorW500(12.0),),
-              ],
             ),
-          ),
-          const SizedBox(height: 10.0),
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 10.0),
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                Text(
-                  '${Strings.followers} 0',
-                  style: MainStyles.kBlackColorW500(12.0),
-                ),
-                Text('${Strings.following} 0', style: MainStyles.kBlackColorW500(12.0),),
-              ],
+            const SizedBox(height: 10.0),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 10.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Text(
+                    '${Strings.followers} ${user.followers ?? ''}',
+                    style: MainStyles.kBlackColorW500(12.0),
+                  ),
+                  Text(
+                    '${Strings.following} ${user.following ?? ''}',
+                    style: MainStyles.kBlackColorW500(12.0),
+                  ),
+                ],
+              ),
             ),
-          ),
-        ],
-      ),
-    );
+          ],
+        ),
+      );
+    }
+
+    return profileBlockWidget;
```

Comment:
Remember this - if your widget is more than 100 lines of code then probably you need to reduce it (break it into smaller pieces)